ARCs/arc-0026.md

```diff
@@ -99,12 +120,20 @@ Request 150 units of Asset ID 45 from an address
 algorand://TMTAD6N22HCS2LKH7677L2KFLT3PAQWY6M4JFQFXQS32ECBFC23F57RYX4?amount=150&asset=45
 ```
 
+call claim(uint64,uint64)byte[] method on contract 11111111
+
+```
+algorand://TMTAD6N22HCS2LKH7677L2KFLT3PAQWY6M4JFQFXQS32ECBFC23F57RYX4?app=11111111&method=claim(uint64,uint64)byte[]&arg=20000&arg=474567&asset=45
+```
+
 ## Rationale
 
+The addition of ARC4 support to ARC26 is a mandatory step for effective interactivity of Algorand operations with users and increases interoperability amongst users as well
+
```

Comment:
I think this section should focus more on the general and broader "Rationale" of the design choices rather than just the ARC4 addition.